Makefile

```diff
@@ -82,7 +82,7 @@ bin/$(ARCH)/$(BIN): build-dirs
 	    -v $$(pwd)/.go:/go                                                 \
 	    -v $$(pwd):/go/src/$(PKG)                                          \
 	    -v $$(pwd)/bin/$(ARCH):/go/bin                                     \
-	    -v $$(pwd)/bin/$(ARCH):/go/bin/$$(go env GOOS)_$(ARCH)             \
+	    -v $$(pwd)/bin/$(ARCH):/go/bin/linux_$(ARCH)                       \
```

Comment:
Oh, I read the patch backwards; I was reading it that Tim had hard-coded it to assume Linux, and that you were changing it to consult _go env_. I can't say why I read it backwards, other than that the commit message makes it sound like we're removing an assumption by interpolating from the environment instead.